src/test/java/space/obminyashka/items_exchange/controller/UserControllerIntegrationTest.java

```diff
@@ -140,6 +140,16 @@ void updateUserPassword_WhenOldPasswordWrong_ShouldThrowInvalidDtoException() th
         assertTrue(mvcResult.getResponse().getContentAsString().contains(getMessageSource("incorrect.password")));
     }
 
+    @Test
+    @WithMockUser(username = "user")
```

Comment:
The interface WithMockUser has its default values:

```
public @interface WithMockUser {
    String value() default "user";

    String username() default "";

    String[] roles() default {"USER"};

    String[] authorities() default {};

    String password() default "password";
```
So, you can omit using the same username by simply using the annotation with default settings.